qcut/scripts/backup-nextjs-pages.sh

```
@@ -0,0 +1,94 @@
+
+# Create backup of Next.js app directory
+if [ -d "apps/web/src/app" ]; then
+    echo "🔄 Backing up apps/web/src/app/ directory..."
+    tar -czf "$BACKUP_FILE" apps/web/src/app/
+    
+    if [ $? -eq 0 ]; then
+        echo "✅ Next.js pages backed up to: $BACKUP_FILE"
+        echo "📊 Backup size: $(du -h "$BACKUP_FILE" | cut -f1)"
+        echo "📁 Files backed up:"
+        tar -tzf "$BACKUP_FILE" | head -20
+        if [ $(tar -tzf "$BACKUP_FILE" | wc -l) -gt 20 ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)